README.md

```diff
@@ -14,15 +15,17 @@ More user relevant information about the firmware are on the wiki page at: https
 For questions write a mail to <berlin@berlin.freifunk.net> or come to our weekly meetings.
 If you find bugs please report them at: https://github.com/freifunk-berlin/firmware/issues
 
+
 ## Development
 
 ### Info
 
-For the Berlin Freifunk firmware we use vanilla OpenWrt with additional patches
-and packages. The Makefile automates firmware
-creation and apply patches / integrates custom freifunk packages. All custom
-patches are located in *patches/* and all additional packages can be found at
-http://github.com/freifunk-berlin/packages_berlin.
+How does this stuff work? 
+
+The idea is to download OpenWrt via git, patch it with all patches in the `patches/` folder and configure it according to the configuration snippets in `configs/`. 
+Then use the OpenWrt buildsystem to build all packages and the Imagebuilder and use the latter to assemble the final image with lists of packages in packages for the router.
+All additional packages can be found at https://github.com/freifunk-berlin/firmware-packages.
```

Comment:
Just read the sentence .... Probably better to write "All Berlin.Freifunk specific packages can be found...".
As we currently include the Gluon-packages feed and the freifunk-packages feed. Probably other feeds (e.g. freifunk-Potsdam) will also be present one time.

Reply:
added.